src/routes/login.svelte

```diff
@@ -131,6 +154,15 @@
         have a keyfile, you can get one by creating an <a
           href="https://www.arweave.org/wallet">Arweave Wallet</a>.
       </p>
+      <div class="weaveid-login">
+        {#if !!weaveIdClient}
+          <button class="weave-id-button" on:click={openWeaveIdLoginModal}>
+            <img src={weaveidSVG} alt="weaveid" /> WeaveID
```

Comment:
```suggestion
            <img src={weaveidSVG} alt="weaveid" /> WeaveID Login
```

Reply:
Would be interested in hearing @maximousblk's thoughts on this.

Reply:
imo go with `WeaveID` or `use WeaveID`

Reply:
Ahh, I like `use WeaveID`. Thanks!